### PR TITLE
Fixed errors related to  punctuation, capitalization, and misspelling

### PR DESCRIPTION
**Pull request by thakurdiwakar**
## Related Issue(s)

#42 

## Proposed Changes

Misspelling of "responsiveness" as "responsivenes."
Incorrect capitalization of "JS" in "Node JS 18.16.0 LTS" instead of "Node.js."
Incorrect capitalization of "ReadMe" instead of "README."

## Type of Change

- [ ] New Feature (Landing Page)
- [ ] New Feature (App)
- [ ] Bug (Landing Page)
- [ ] Bug (App)
- [ ] New Component (App)
- [x] Enhancement
- [x] Documentation / ReadMe / Workflows
- Other (If not in above options):  

## Your Idea for Level (GSSoC)

- [ ] Level 1
- [ ] Level 2
- [ ] Level 3

### CODE REVIEW

File: ReadMe.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ Here’s a recommended guide, but if you want to know what exactly I use, feel f
 * **Operating System:** Windows / Mac / Linux
 * **Version Control:** Git
 * **Browser:** Google Chrome + React Developer Tools
-* **Run Time:** Node JS 18.16.0 LTS
+* **Run Time:** Node.js 18.16.0 LTS
```
Comment (praveenscience) [inline ReadMe.md:47]: We want to say it as Node JS. 😊

File: ReadMe.md [hunk 2]
```diff
@@ -33,7 +33,7 @@ Here's a list of all the features of our GitHub Profile ReadMe customisation too
 * ⏰ Saves time by providing all available components and customisation options in one place.
 * 💪🏻 Helps developers create high-quality and informative GitHub Profile ReadMe pages that showcase their skills and experience.
 
-With our GitHub profile ReadMe customisation tool, you can take your GitHub profile to the next level and impress potential employers and collaborators. Try it out today and see the difference it can make! 🤗
+With our GitHub profile README customisation tool, you can take your GitHub profile to the next level and impress potential employers and collaborators. Try it out today and see the difference it can make! 🤗
```
Comment (praveenscience) [inline ReadMe.md:36]: Make this all as "GitHub Profile ReadMe" - It's the product name! 💪

File: ReadMe.md [hunk 3]
```diff
@@ -67,7 +67,7 @@ $ npm start
 
 If `npm ci` gives issues on Windows or Linux machines, please use `npm i` (or if you’re not lazy, run `npm install`).
 
-If you’re facing any issues, feel free to message me on WhatsApp / LinkedIn / Twitter / Discord / Email (on the order of responsivenes, email being the slowest)! 😅
+If you’re facing any issues, feel free to message me on WhatsApp / LinkedIn / Twitter / Discord / Email (on the order of responsiveness, email being the slowest)! 😅
```
Comment (praveenscience) [inline ReadMe.md:70]: Dang! Sorry! 😢

File: ReadMe.md [hunk 4]
```diff
@@ -91,8 +91,8 @@ Here are some quick and simple instructions, which no one can mess up! 💪🏻
    2. Install all the dependencies and issue the start command.
 4. Run the project and explore every bit of the project. Here’s a [walkthrough](https://www.youtube.com/praveenscience?sub_confirmation=1) video! 📺
 5. Understand the already raised - accepted issues to start work with. 🤔
-   1. If the issue is already assigned to someone else, please enter your comment but look through and try to see how you can fit.
-   2. If it’s not already assigned, tell how you can solve this issue, explain your procedure in your mind and ask to assign.
+   1. If the issue is already assigned to someone else, please enter your comment but read through and try to see how you can fit.
```
Comment (praveenscience) [inline ReadMe.md:94]: I believe look is better here? Or not?

File: ReadMe.md [hunk 5]
```diff
@@ -91,8 +91,8 @@ Here are some quick and simple instructions, which no one can mess up! 💪🏻
...
+   2. If it’s not already assigned, describe how you can solve this issue, explain your procedure in your mind and ask to assign.
```
Comment (praveenscience) [inline ReadMe.md:95]: Nice one, thanks. 😊

**Review by praveenscience — CHANGES_REQUESTED** [review]
Very very small stuff...

**Review by praveenscience — CHANGES_REQUESTED** [review]
Very very small stuff...

File: ReadMe.md (hunk 2 above)
Comment (thakurdiwakar) [reply ReadMe.md:36]: Okay

File: ReadMe.md (hunk 3 above)
Comment (thakurdiwakar) [reply ReadMe.md:70]: Its okay 😊

File: ReadMe.md (hunk 5 above)
Comment (thakurdiwakar) [reply ReadMe.md:95]: Your welcome

File: ReadMe.md (hunk 4 above)
Comment (thakurdiwakar) [reply ReadMe.md:94]: I think look's better now

**Comment by thakurdiwakar** [comment]
Could you please check if there are any other changes needed?

File: ReadMe.md (hunk 1 above)
Comment (praveenscience) [reply ReadMe.md:47]: Let's fix this and we'll finish this off?

Comment (thakurdiwakar) [reply ReadMe.md]: Done,please check

**Review by praveenscience — APPROVED** [review]
👍

**Review by praveenscience — APPROVED** [review]
👍

**Comment by praveenscience** [comment]
Can you resolve the merge conflicts and push? Thanks man. Let's get this merged! 💪

**Comment by thakurdiwakar** [comment]
I have resolved ,please check

**Comment by thakurdiwakar** [comment]
When will it be merged? @praveenscience

**Comment by praveenscience** [comment]
image

**Comment by praveenscience** [comment]
Sorry, just checked, it still has Merge Conflicts I cannot resolve.

**Comment by thakurdiwakar** [comment]
Screenshot_2023-06-06-09-53-20-821_com android chrome-edit


See this , there is no option

**Comment by praveenscience** [comment]
Don't you have a local branch? Did you not pull it locally and worked? 🤔

**Comment by thakurdiwakar** [comment]
No, what can i do now?

**Pull request closed without merging**